src/theme/foundations/typography.ts

```diff
@@ -174,7 +174,6 @@ const typography: Typography = {
         subtitle2: fontSizeValues.mobile["subtitle2"],
       },
     },
-    // deprecated semantic tokens
```

Comment:
I see now that I was mistaken. Everything below this comment should be removed. The reasoning is that if there are tokens in the `desktop` and `mobile` objects that mirror tokens at the root of the `fontSizes` object, then the tokens at the root should be removed. I'm sorry for the double work. I must not have looked at the full block of code when I first commented about the removal of the `breadcrumbs` tokens. :(

Reply:
This this true only for `breadcrumbs` or also remove `button` and `helper`?

Reply:
That would be applicable to all tokens below that comment: `breadcrumbs`, `button`, and `helper`.

Reply:
Ok, removed.